t/free.t

```diff
@@ -0,0 +1,14 @@
+#!perl -w
+
+use Test::More tests => 1;
+
+use HTML::Parser;
+$p = new HTML::Parser(
```

Comment:
Please don't use indirect object notation or -w
```perl
#!perl
use strict;
use warnings;
use HTML::Parser ();
use Test::More tests => 1;

my $p = HTML::Parser->new( ...
```

Reply:
both fixed, I blindly copy/paste the test from sprout :-)
I've enforce strict/warnings